### DISCUSSION TITLE
Edge router

### DISCUSSION THREAD

**Discussion by WolfVerrron18** [antvis/G6#6680] [Show and tell]
Good afternoon, please tell me if it is possible to implement the edge style as in x6 type manhattan

**Comment by petercat-assistant** [comment]
In the latest version of G6 (5.0), you can implement custom edge styles, including styles similar to the Manhattan routing style found in X6. G6 provides flexibility in defining edge shapes and routing strategies, allowing you to create edges that follow a grid-like path, similar to the Manhattan style.

To achieve this, you can use G6's built-in functionalities to define custom edge shapes and control the routing logic. You might need to write some custom logic to calculate the path points based on your grid layout.

If you need further assistance or a specific example, feel free to ask!

**Reply by WolfVerrron18** [reply]
we need a concrete example of the implementation of behavior from x6 to g6
